### PR TITLE
HAWQ-1274. Add apache rat check in travis CI.

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @huor @paul-guo-

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +39,7 @@ script:
   - make -j$(sysctl -n hw.ncpu)
   - make feature-test
   - make feature-test-clean
+  - mvn apache-rat:check
```
Comment (stanlyxiang) [inline .travis.yml]: what about move rat check before make hawq. since if rat check fail, no need to build at all.

Comment (xunzhang) [inline .travis.yml]: totally agree, good advice!

**Comment by stanlyxiang** [comment]
LGTM , +1

**Comment by paul-guo-** [comment]
+1

**Comment by xunzhang** [comment]
Merged into master.

**Comment by huor** [comment]
Here is the jenkins project (https://builds.apache.org/job/HAWQ-rat/) that do the rat check as well which is added by @edespino recently. It also checks the year of the copyright in NOTICE and hawq version in pom.xml. Any idea to unify the travis and jenkins project for rat?

**Comment by xunzhang** [comment]
Who is now taking charge of Jenkins CI? I think for apache project, if you can refine our Jenkins CI script, it might be better choice.

**Comment by huor** [comment]
I think all committer have the access to add jenkins project and configure them. @edespino, please share more on this.

**Pull request closed without merging**